src/index.js

```diff
@@ -0,0 +1,50 @@
+import './style.css';
+import UI from './modules/UI.js';
+import Store from './modules/localstorage.js';
+import Todostatus from './modules/toDoStatus.js';
+
+// Event for Displaying the to do tasks
+document.addEventListener('DOMContentLoaded', UI.displayToDoList);
+
+// Event for adding new to do tasks
+document.querySelector('#todo-form').addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    e.preventDefault();
+    UI.getNewToDoTask();
+  }
+});
+
+// Event for deleting new item for the list
```

Comment:
- [ ] Great effort working on your project, it looks really nice on the browser, But I would like to draw your attention to something very important. 
- The main goal of this milestone you are currently working on i.e `To Do list: list structure`, is to have a list of some simple todo tasks creating them using an `array of an object` and with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\].

Possible Example 👇 

```JS
const tasks = [
  {
    description: 'An example described in the string',
    completed: false,
    index: 1,
  },
  {
    description: 'An example described in the string',
    completed: false,
    index: 2,
  },
  {
    description: 'An example described in the string',
    completed: false,
    index: 3,
  },
];
```